pkgs/servers/gerbera/default.nix

```diff
@@ -0,0 +1,79 @@
+{ stdenv, fetchFromGitHub
+, cmake, pkg-config
+# required
+, libupnp, libuuid, pugixml, libiconv, sqlite, zlib, spdlog, fmt
+, pkgs
+# options
+, enableDuktape ? true
+, enableCurl ? true
+, enableTaglib ? true
+, enableLibmagic ? true
+, enableLibmatroska ? true
+, enableAvcodec ? false
+, enableLibexif ? true
+, enableExiv2 ? false
+, enableFFmpegThumbnailer ? false
+, enableInotifyTools ? true
+}:
+
+with stdenv.lib;
+let
+  optionOnOff = option: if option then "on" else "off";
+in stdenv.mkDerivation rec {
+  pname = "gerbera";
+  version = "1.6.1";
+
+  src = fetchFromGitHub {
+    repo = pname;
+    owner = pname;
+    rev = "v${version}";
+    sha256 = "sha256:05ca27r9sidbl7xns9hcdan8wgjrpg26n1wq1vp247c9bqhpyql8";
+  };
+
+  cmakeFlags = [
+    "-DWITH_JS=${optionOnOff enableDuktape}"
+    "-DWITH_CURL=${optionOnOff enableCurl}"
+    "-DWITH_TAGLIB=${optionOnOff enableTaglib}"
+    "-DWITH_MAGIC=${optionOnOff enableLibmagic}"
+    "-DWITH_MATROSKA=${optionOnOff enableLibmatroska}"
+    "-DWITH_AVCODEC=${optionOnOff enableAvcodec}"
+    "-DWITH_EXIF=${optionOnOff enableLibexif}"
+    "-DWITH_EXIV2=${optionOnOff enableExiv2}"
+    "-DWITH_FFMPEGTHUMBNAILER=${optionOnOff enableFFmpegThumbnailer}"
+    "-DWITH_INOTIFY=${optionOnOff enableInotifyTools}"
+    # systemd service will be generated alongside the service
+    "-DWITH_SYSTEMD=OFF"
+  ];
+
+  nativeBuildInputs = [ cmake pkg-config ];
+
+  buildInputs = [
+    libupnp libuuid pugixml libiconv sqlite zlib fmt.dev
+    spdlog
+  ]
+  ++ optionals enableDuktape [ pkgs.duktape ]
+  ++ optionals enableCurl [ pkgs.curl ]
+  ++ optionals enableTaglib [ pkgs.taglib ]
+  ++ optionals enableLibmagic [ pkgs.file ]
+  ++ optionals enableLibmatroska [ pkgs.libmatroska pkgs.libebml ]
+  ++ optionals enableAvcodec [ pkgs.libav.dev ]
+  ++ optionals enableLibexif [ pkgs.libexif ]
+  ++ optionals enableExiv2 [ pkgs.exiv2 ]
+  ++ optionals enableInotifyTools [ pkgs.inotify-tools ]
+  ++ optionals enableFFmpegThumbnailer [ pkgs.ffmpegthumbnailer ];
+
+
+  meta = with stdenv.lib; {
+    homepage = https://docs.gerbera.io/;
+    repositories.git = "https://github.com/gerbera/gerbera";
```

Comment:
I think you can just remove it. People can always use `pkg.src.homepage` if they need this information.